ponyville/block-templates/index.html

```diff
@@ -0,0 +1,7 @@
+<!-- wp:query -->
+	<!-- wp:query-loop -->
+		<!-- wp:post-title {"isLink":true} /-->
+	<!-- /wp:query-loop -->
+<!-- /wp:query -->
+
+<!-- wp:post-content /-->
```

Comment:
Do we want to show the FSE to the users with this theme? Or should we insert the templates using `do_blocks()` ?

Reply:
I wish do_blocks() was fully featured enough. :(
* There are still a lot of blocks that can't be created using this method, they are limited to the block templates only (if I understand correctly) so there is much we won't be able to do.
* Template Parts don't work with do_blocks(). :(

If both of those things weren't true then we could I think.

However, I would argue that instead Themes should have control as to which "management" options should be available when using it.  Rather than FSE/Customizer being enabled/disabled with the presence of a /block-templates folder I think a better option would be to give themes a theme_supports() option (configurable via a call to theme_supports() or via theme.json) to enable/disable FSE/Customizer.  (Which... I will continue to argue that those things shouldn't be mutually exclusive).

Reply:
Yes, 100% with you on that. I think we should focus on getting that before we can actually launch any "Ponytheme" then since we don't really want the editor to show until it's really ready for general use.